share/models/change.py

```diff
@@ -271,3 +242,66 @@ def _resolve_change(self):
             else:
                 change[k] = v
         return change
+
+    def _merge_objects(self, from_obj, into_obj):
+        from share.models import ShareObject
+
+        # TODO less hacky way of expressing what can be merged
+        MERGEABLE_MODELS = {'AbstractCreativeWork', 'AbstractAgent', 'Award', 'AbstractWorkRelation', 'AbstractAgentRelation', 'AbstractAgentWorkRelation', 'ThroughTags', 'ThroughSubjects', 'ThroughContributor', 'ThroughAwards'}
+        concrete_model = from_obj._meta.concrete_model
+        if concrete_model.__name__ not in MERGEABLE_MODELS:
+            raise InvalidMergeError('Invalid model for merging: %s' % concrete_model)
```

Comment:
This is an odd inconsistency, but use new style formatting on exceptions.

The only case for old style format is log messages. There may be additional compute time required to format the string that can be avoided if the log is too high/low of a level.